### PR TITLE
Update screenshot errors inbox

### PR DESCRIPTION
**Pull request by akristen**
This updates an errors inbox doc to address suggestions from jira ticket

### CODE REVIEW

File: src/content/docs/errors-inbox/error-users-impacted.mdx [hunk 1]
```diff
@@ -6,58 +6,59 @@ redirects:
...
+In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents captures user-identifying metadata at the time of ingest. 
 
-## Attribute conventions to capture users impacted [#attributes]
+For cases where you need to add a custom attribute, we recommend reviewing [OpenTelemetry's attribute conventions](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction) because there are multiple common conventions for naming end user attributes.
 
-There are multiple common conventions for naming end user attributes.
-First, we'll check for `enduser.id` according to the [OpenTelemetry standard](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes). Then we'll use the following attributes in order of priority:
+For example, one [OpenTelemetry standard](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes) uses `enduser.id`. We'll use the following attributes in order of priority:
```
Comment (akristen) [inline src/content/docs/errors-inbox/error-users-impacted.mdx:26]:
```suggestion
For example, [OpenTelemetry standards](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes) uses the following attributes in order of priority:
```

File: src/content/docs/errors-inbox/error-users-impacted.mdx [hunk 2]
```diff
@@ -6,58 +6,59 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-You can now see the number of users impacted by an error group which allows you to focus your effort on the
-highest impacting error groups. Users impacted are listed next to the occurrences metric in the list view.
-You can also [set up Alerts](/docs/errors-inbox/error-users-impacted/#alerting-on-users-impacted) for error groups impacting more than a certain number of users.
+New Relic agents automatically capture data about user impact.When you organize by error group, you can see how many users are affected by errors. This helps you prioritize where to focus your debugging efforts by showing you the errors with the highest numbers of users affected.
```
Comment (akristen) [inline src/content/docs/errors-inbox/error-users-impacted.mdx:9]:
```suggestion
New Relic agents automatically capture data about user impact. When you organize by error group, you can see how many users are affected by errors. This helps you prioritize where to focus your debugging efforts by showing you the errors with the highest numbers of users affected.
```

File: src/content/docs/errors-inbox/error-users-impacted.mdx [hunk 3]
```diff
@@ -6,58 +6,59 @@ redirects:
...
+</figcaption>
 
-## Getting started [#getting-started]
+## Custom attributes to track users impacted  [#custom-attributes]
 
-User impact metrics are collected automatically if your error events have an attribute identifying the end user.
-For agents that don’t natively support adding a user identifier to telemetry, we recommend adding a custom attribute and following [OpenTelemetry's](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction) convention attribute for identity attributes, which is `enduser.id`. The process of sending custom attributes to New Relic is detailed [here](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom).
+In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents captures user-identifying metadata at the time of ingest.
```
Comment (cbehera-newrelic) [inline src/content/docs/errors-inbox/error-users-impacted.mdx:22]: You can manually add custom attributes so our agents capture .....

File: src/content/docs/errors-inbox/error-users-impacted.mdx [hunk 4]
```diff
@@ -6,58 +6,59 @@ redirects:
...
-First, we'll check for `enduser.id` according to the [OpenTelemetry standard](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes). Then we'll use the following attributes in order of priority:
+For example, one [OpenTelemetry standard](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes) uses `enduser.id`. We'll use the following attributes in order of priority:
 
-* `userId`
-* `user`
+    * `userId`
+    * `user`
 
-### Querying for users impacted [#alerts]
+You can review how New Relic processes custom attributes by reviewing our [Collect custom events doc](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom).
```
Comment (cbehera-newrelic) [inline src/content/docs/errors-inbox/error-users-impacted.mdx:31]: ...reviewing our [collect custom events doc] ......

File: src/content/docs/errors-inbox/error-users-impacted.mdx (hunk 3 above)
Comment (akristen) [inline src/content/docs/errors-inbox/error-users-impacted.mdx:22]:
```suggestion
In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents capture user-identifying metadata at the time of ingest. 
```

**Review by cbehera-newrelic — APPROVED** [review]
LGTM

**Pull request merged**